libcrux-ml-dsa/src/arithmetic.rs

```diff
@@ -10,17 +10,60 @@ impl PolynomialRingElement {
         // FIXME: hax issue, 256 is COEFFICIENTS_IN_RING_ELEMENT
         coefficients: [0i32; 256],
     };
+
+    pub(crate) fn add(&self, rhs: &Self) -> Self {
```

Comment:
Can all these small functions get `inline`s? Just to make sure the compiler is doing the right thing.